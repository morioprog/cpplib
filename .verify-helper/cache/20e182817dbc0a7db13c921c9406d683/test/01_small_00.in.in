12
2 1 2
2 3 4
2 8 9
0
3 5 6 7
0
2 10 11
0
0
0
0
0
9
0 4 3
0 6 2
1 11
1 6
1 4
0 4 2
1 10
1 3
1 0
