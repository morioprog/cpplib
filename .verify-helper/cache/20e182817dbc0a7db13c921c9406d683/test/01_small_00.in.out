5
5
3
7
0
0
